.github/workflows/release-test.yml

```diff
@@ -29,6 +29,24 @@ jobs:
       - name: super-test
         run: |
             ./super-test.sh ${{ matrix.compiler }}
+  ManyLinux:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - cross-compiler: manylinux2014-x64
+          - cross-compiler: manylinux2014-x86
+            docker-run-args: --platform linux/386
+    steps:
+      - uses: actions/checkout@v2
+      - name: install dockcross
+        run: |
+            docker run ${{ matrix.docker-run-args }} --rm dockcross/${{ matrix.cross-compiler }} > ./dockcross
+            chmod +x ./dockcross
+      - name: super-test
+        run: |
+            ./dockcross env ENABLE_32BIT_TEST=OFF ENABLE_VALGRIND_TEST=OFF ./super-test.sh g++
```

Comment:
Are the non-quick tests actually valuable to run for ManyLinux? I think we only really care about the configuration that will be used by the Python module, right? So e.g. testing with no-exceptions, no-rtti, etc. isn't needed.

Reply:
The quick tests didn't catch the original GLIBC issue, but the non-quick tests did.

FYI: I'm actually using capnproto with OCaml, not Python. The Python manylinux is just a convenient and somewhat well-known target to compile against. But I'm quite fine with restricting it to just the Python/C interop flags; that would still be testing GLIBC so that helps out everybody.

I'll skip the no-exceptions/no-rtti checks before the PR is closed.

Thanks for a great product!!

(This PR can wait; not blocking)

Reply:
Which glibc issue do you mean? The SYS_renameat2 issue? That should be covered by the quick tests...

Reply:
> Which glibc issue do you mean? The SYS_renameat2 issue?

Yep

> That should be covered by the quick tests...

Ack, you are correct; I must have had some intermediate files lying around. I'll downgrade this PR to just do the quick test. Thanks for catching.